openslides/utils/autoupdate.py

```diff
@@ -393,15 +41,20 @@ def inform_changed_data(instances: Union[Iterable[Model], Model], information: D
             pass
 
     # Put all collection elements into the autoupdate_bundle.
+    collection_elements = {}
```

Comment:
The comment in line 43 is not correct. Can you please rewrite it? Please also extend the docstring because we do not use the AutoupdateBundleMiddleware in all cases now, do we?

Reply:
done